core-services/prow/02_config/_plugins.yaml

```diff
@@ -256,27 +256,6 @@ cherry_pick_unapproved:
   comment: This PR is not for the master branch but does not have the `cherry-pick-approved`  label.
     Adding the `do-not-merge/cherry-pick-not-approved`  label.
 config_updater:
-  cluster_groups:
```

Comment:
The change on this file needs to be recovered.

Reply:
Thanks, restored `core-services/prow/02_config/_plugins.yaml` and rebased in the latest commit.